docs/dictionary/en-custom.txt

```diff
@@ -33,6 +33,7 @@ Kubernetes
 kubectl
 bitnami
 postgresql
+postgres
```

Comment:
I think `postgresql` is enough

Reply:
do we need  postgresql  and postgres?

Reply:
Its the same thing.
postgresql is the full name of postgres